other/upgrade-helper.php

```diff
@@ -457,3 +458,27 @@ function array_column($input, $column_key, $index_key = null)
 		return $arr;
 	}
 }
+
+/**
+ * Creates the json_encoded array for the current cache option.
+ *
+ * @return string a json_encoded array with the selected API options
```

Comment:
Where is the JSON?

Reply:
There is not json entry anymore, it still is a plain string I just forgot to update the dock block